tools/compile-schedule.js

```diff
@@ -0,0 +1,5 @@
+import fs from 'fs/promises';
+
+const json = await fs.readFile('./schedule.json', { encoding: 'utf8' });
+
+await fs.writeFile('./src/schedule.js', `export default ${json}`);
```

Comment:
Is this what makes the `schedule.json` from upstream node into something that Renovate bot can use in the main repo?

Reply:
currently i use only the local `schedule.json` which can  be manuallly updated with `yarn get-latest-schedule-curl`.

In a future pr we can do some kind of auto updatting this file to trigger a ci release.